### PR TITLE
Add app store links to download

### PR DESCRIPTION
**Pull request by joaoqalves**
I don't know much about Node / (P)react / NodeJS :)

### CODE REVIEW

File: src/routes/city/index.js [hunk 1]
```diff
@@ -88,13 +90,13 @@ class City extends Component {
...
 
-                <Flex row>
+                <Alternative row>
                   {city.company
                     .filter(item => item !== 'uber')
                     .map(c => (
-                      <Company>{c}</Company>
-                    ))}
-                </Flex>
+                    <Company href={getAppStoreLink(c.company)} target="_blank">{c.company}</Company>
```
[the remark this answers is not in the document]
Comment (SaraVieira) [reply src/routes/city/index.js]: Its something to do with the node version. You can delete osmosis as a dependency for now and install again

It should work

**Comment by SaraVieira** [comment]
In this line:
```
export const getAppStoreLink = (app) => storeLinks[app.toLowerCase() || defaultApp][mobileOs];
```

You just need: 
```
export const getAppStoreLink = (app) => storeLinks[app.toLowerCase() || 'default'][mobileOs];
```

👍

**Comment by joaoqalves** [comment]
@SaraVieira thx! Shouldn't it be `(app || 'default').toLowercase()` though?

**Comment by SaraVieira** [comment]
YES! Sorry, my code interpeter ahah

Also can you run it now if delete osmosis ?

**Comment by joaoqalves** [comment]
@SaraVieira It does run :) If it works, should I squash these commits?

**Comment by SaraVieira** [comment]
No need, I can sqash on merge :)

It does recognize my phone as ios :(

**Comment by joaoqalves** [comment]
I've misused `mobile-detect`'s API. Should be OK in the next build

**Comment by SaraVieira** [comment]
Works 💯 !

Merged! Thank you so much for handling all the edge cases :)

**Pull request merged**